### DISCUSSION TITLE
Support for more web crawlers

### DISCUSSION THREAD

**Discussion by zero734kr** [vercel/next.js#40690] [Ideas]
### Describe the feature you'd like to request

Hello,

I wonder if it would support korean search engines and social media services like Naver, Daum and KakaoTalk because I was reading that even using `fallback: true`, for web crawlers like google it would use `fallback: 'blocking'` automatically. ([Reference](https://nextjs.org/docs/api-reference/data-fetching/get-static-paths))

I was curious if it supports, but apparently not because I couldn't find it [here](https://github.com/vercel/next.js/blob/76083210eded33932f46ffa1cf538515200bca50/packages/next/server/web/spec-extension/user-agent.ts#L29) nor [from here](https://github.com/vercel/next.js/blob/v12.3.1-canary.3/packages/next/shared/lib/router/utils/is-bot.ts).

According to InterAd, the statistic on search engine market share in south korea is like [this](https://www.interad.com/en/category/insights/korean-search-engine-market-share.html)

I think Naver, Daum and KakaoTalk are the most important, because the first two are search engines and KakaoTalk is in first place in [instant messenger market share](https://www.insiderintelligence.com/content/kakaotalk-most-popular-messaging-app-south-korea-by-massive-margin) (and it supports og crawling)
But if you want consider other search engines, Nate is using the same search engine as Daum according to [this article](https://www.fnnews.com/news/201401121718111591). (Sorry, I couldn't find the english version)

User agent headers of Naver (main keywords: `Yeti` and `NaverBot`): 
* https://searchadvisor.naver.com/guide/seo-basic-firewall
* https://developers.whatismybrowser.com/useragents/explore/software_name/naver-com-crawler/

User agent headers of Daum (main keyword: `Daum`):
* https://cs.daum.net/m/faq/faqlist/36378
* https://developers.whatismybrowser.com/useragents/explore/software_name/daum/

User agent headers of KakaoTalk (main keywords: `KAKAOTALK`, `kakaotalk-scrap`, `kakaostory-og-reader`):
* https://devtalk.kakao.com/t/scrap-url/116202
* https://whatmyuseragent.com/apps/kakaotalk

### Describe the solution you'd like

Edit regex to detect more web crawlers

### Describe alternatives you've considered

Something like next.config.js option to customize isBot regex?

**Comment by TylerShin** [comment]
👍

**Comment by meotimdihia** [comment]
We should support social news sharing, It is essential.
